odoo/service/security.py

```diff
@@ -24,5 +17,5 @@ def check_session(session, env):
     expected = self._compute_session_token(session.sid)
     if expected and odoo.tools.misc.consteq(expected, session.session_token):
```

Comment:
Shouldn't we check the `pre_session_token` if the MFA url is set and the `session_token` if not?

Reply:
No, we don't want to risk a "pre-session" to be considered a valid session for any situation other than submitting an MFA validation code.